consensus/mint-client/src/bin/main.rs

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) 2018-2022 The MobileCoin Foundation
+
+//! Entrypoint for the consensus mint client.
+
+use clap::Parser;
+use grpcio::{ChannelBuilder, EnvBuilder};
+use mc_common::logger::{create_app_logger, o};
+use mc_consensus_api::consensus_client_grpc::ConsensusClientApiClient;
+use mc_consensus_mint_client::Config;
+use mc_crypto_keys::{Ed25519Pair, Signer};
+use mc_crypto_multisig::{MultiSig, SignerSet};
+use mc_transaction_core::mint::{
+    constants::NONCE_LENGTH, MintConfig, MintConfigTx, MintConfigTxPrefix,
+};
+use mc_util_grpc::ConnectionUriGrpcioChannel;
+use rand::{rngs::StdRng, RngCore, SeedableRng};
+use std::sync::Arc;
+
+fn main() {
+    let (logger, _global_logger_guard) = create_app_logger(o!());
+    let config = Config::parse();
+
+    let env = Arc::new(EnvBuilder::new().name_prefix("mint-client-grpc").build());
+    let ch = ChannelBuilder::default_channel_builder(env).connect_to_uri(&config.node, &logger);
+    let api_client = ConsensusClientApiClient::new(ch);
+
+    let signer_1 = Ed25519Pair::from(config.private_key);
+
+    let mut rng: StdRng = SeedableRng::from_seed([1u8; 32]);
```

Comment:
Seeding this looks like it will cause the client to use the same nonce every time it runs

Reply:
Yes, that was the idea - I wanted to test that the nonce checking works. This client is very much WIP.